azure-iot-device/tests/provisioning/pipeline/test_provisioning_pipeline.py

```diff
@@ -1,525 +1,525 @@
-        self, mock_provisioning_pipeline, mock_mqtt_transport
-    ):
-        mock_provisioning_pipeline.connect()
-        mock_mqtt_transport.on_mqtt_connected_handler()
-        mock_provisioning_pipeline.wait_for_on_connected_to_be_called()
-        mock_provisioning_pipeline.disconnect()
-
-        mock_mqtt_transport.disconnect.assert_called_once_with()
-
-    @pytest.mark.it("Is ignored if already disconnected")
-    def test_disconnect_ignored_if_already_disconnected(
-        self, mock_provisioning_pipeline, mock_mqtt_transport
-    ):
-        mock_provisioning_pipeline.disconnect(None)
-
-        mock_mqtt_transport.disconnect.assert_not_called()
-
-    @pytest.mark.it("After complete calls handler with `disconnected` state")
-    def test_disconnect_calls_client_disconnect_callback(
-        self, mock_provisioning_pipeline, mock_mqtt_transport
-    ):
-        mock_provisioning_pipeline.connect()
-        mock_mqtt_transport.on_mqtt_connected_handler()
-        mock_provisioning_pipeline.wait_for_on_connected_to_be_called()
-
-        mock_provisioning_pipeline.disconnect()
-        mock_mqtt_transport.on_mqtt_disconnected_handler(None)
-
-        mock_provisioning_pipeline.wait_for_on_disconnected_to_be_called()
-        mock_provisioning_pipeline.on_disconnected.assert_called_once_with("disconnected")
-
-
-@pytest.mark.parametrize("params_security_clients", different_security_clients)
-@pytest.mark.describe("Provisioning pipeline - Enable")
-class TestEnable(object):
-    @pytest.mark.it("Calls subscribe on provider")
-    def test_subscribe_calls_subscribe_on_provider(
-        self, mock_provisioning_pipeline, mock_mqtt_transport
-    ):
-        mock_provisioning_pipeline.connect()
-        mock_mqtt_transport.on_mqtt_connected_handler()
-        mock_provisioning_pipeline.wait_for_on_connected_to_be_called()
-        mock_provisioning_pipeline.enable_responses()
-
-        assert mock_mqtt_transport.subscribe.call_count == 1
-        assert mock_mqtt_transport.subscribe.call_args[1]["topic"] == fake_sub_unsub_topic
-
-
-@pytest.mark.parametrize("params_security_clients", different_security_clients)
-@pytest.mark.describe("Provisioning pipeline - Disable")
-class TestDisable(object):
-    @pytest.mark.it("Calls unsubscribe on provider")
-    def test_unsubscribe_calls_unsubscribe_on_provider(
-        self, mock_provisioning_pipeline, mock_mqtt_transport
-    ):
-        mock_provisioning_pipeline.connect()
-        mock_mqtt_transport.on_mqtt_connected_handler()
-        mock_provisioning_pipeline.wait_for_on_connected_to_be_called()
-        mock_provisioning_pipeline.disable_responses(None)
-
-        assert mock_mqtt_transport.unsubscribe.call_count == 1
-        assert mock_mqtt_transport.unsubscribe.call_args[1]["topic"] == fake_sub_unsub_topic
+# # -------------------------------------------------------------------------
+# # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Pet peeve/Nit: huge deltas like this make for painful source management (merges, rebases, git blame, etc)  If you want to comment out the tests, just add an underscore to the beginning of the class/test name so pytest doesn't pick it up.

Reply:
Yeah that would have been smarter

---
In reply to: [347139962](https://github.com/Azure/azure-iot-sdk-python/pull/392#discussion_r347139962) [](ancestors = 347139962)